lib/Migrate_ast.ml

```diff
@@ -180,6 +180,8 @@ module Location = struct
 
   let compare_end_col x y = Position.compare_col x.loc_end y.loc_end
 
+  let line_difference fst snd = snd.loc_start.pos_lnum - fst.loc_end.pos_lnum
```

Comment:
This function doesn't make sense if `fst` is after (both start and end positions) `snd`. I think we should at least test `assert (fst.loc_start.pos_lnum <= snd.loc_start.pos_lnum)`.
Also I'm not sure what this function means if `snd` is inside `fst`, we should probably assert that the result is `>= 0` too.
Could you add some documentation about that ?

Reply:
I agree on the principle, but I've just discovered that this invariant doesn't hold anymore when comments have been relocated, so for now I've just added a docstring for this function and I will investigate more deeply this relocation issue in another PR. It doesn't look blocking for this PR as this is what was already done in master (the difference computation) anyway.